src/components/media-player/ha-media-player-browse.ts

```diff
@@ -263,13 +228,19 @@ export class HaMediaPlayerBrowse extends LitElement {
           : ""}
       </div>
       ${this._error
-        ? html`<div class="container error">
-            ${this._renderError(this._error)}
-          </div>`
+        ? html`
+            <div class="container error">
+              ${this._renderError(this._error)}
+            </div>
+          `
         : currentItem.children?.length
-        ? hasExpandableChildren
+        ? mediaClass.layout === "grid"
           ? html`
-              <div class="children">
+              <div
+                class="children ${classMap({
+                  portrait: mediaClass.thumbnail_ratio === "portrait",
+                })}"
+              >
                 ${currentItem.children.map(
                   (child) => html`
```

Comment:
```suggestion
                  (child) => { const classStyles = MediaClassBrowserSettings[child.media_class]; return html`
```

Reply:
Not sure what you are going for here

Reply:
There are no per child classes needed

Reply:
You keep looking up the `MediaClassBrowserSettings`, do it once and store in a var.